apps/client/src/router/index.js

```diff
@@ -86,7 +88,7 @@ router.beforeEach((to) => { // Cf. https://github.com/vueuse/head pour des trans
   document.title = `${specificTitle}${MAIN_TITLE}`
 })
 
-router.beforeEach(async (to, from, next) => {
+router.beforeEach(async (to, _from, next) => {
```

Comment:
Il y a 3 beforeEach, comment le router sait le quelle prendre ?

Reply:
Hello Baudoin, merci pour la review ! Ils sont appelés dans l'ordre d'après la doc : 
https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards
> Global before guards are called in creation order, whenever a navigation is triggered.

Après en terme de bonne pratique je n'ai pas trouvé l'info, est-ce qu'il vaut mieux avoir plusieurs `beforeEach` avec chacun un but spécifique, ou est-ce qu'il vaut mieux en avoir un seul qui gère tout ? Perso je trouve la première option plus maintenable.
@this-is-tobi si tu as une opinion !

Reply:
Tout à fait je vais commenter !